cmd/ecr-credential-provider/main.go

```diff
@@ -60,8 +60,12 @@ type ecrPlugin struct {
 }
 
 func defaultECRProvider(region string) (*ecr.ECR, error) {
+	cfg := aws.Config{}
+	if region != "" {
+		cfg.Region = aws.String(region)
+	}
```

Comment:
I think we need a log warning that no region was found in the image reference/the default region will be used/etc.